src/w3c/headers.js

```diff
@@ -362,11 +362,15 @@ export function run(conf) {
   conf.isTagFinding =
     conf.specStatus === "finding" || conf.specStatus === "draft-finding";
 
-  if (conf.isRecTrack && !hasGitHubIssuesLink(conf)) {
-    const msg = docLink`Missing link to GitHub in head of document.`;
-    const hint = docLink`Please add the ${"[github]"} configuration option to document's ${"[respecConfig]"}.`;
-    showError(msg, name, { hint });
+  if (conf.isRecTrack && !conf.github && !conf.wgPublicList) {
+    const msg =
+      "W3C Process requires a either a link to a public repository or mailing list.";
```

Comment:
Nit: using backticks will make it single line 🙈

Reply:
I'm sure they will fix this in next prettier version.